hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -147,13 +192,20 @@ Network boot functionality is included in the 2020-02-13 Raspbian Buster release
 sudo apt update
 sudo apt upgrade
                                                                           
-# Select the stable bootloader release series. The default is 'critical' which is only updated 
-# for major bugs, security fixes or hardware compatibility changes.
-# Network boot is not enabled in the default bootloader.
+# Network boot is not yet enabled in the production bootloader image.
 # As root:-
 echo FIRMWARE_RELEASE_STATUS="stable" > /etc/default/rpi-eeprom-update
 ```
 
+#### Firmware release status
+The firmware release status maps to sub-directory of bootloader firmware images and is used to select different release streams according to stability. By default, Raspbian only selects critical updates (security fixes or major hardware compatiblity changes) since most users do not use alternate boot modes (TFTP, USB etc)
```

Comment:
maps to sub-directory -> maps to a subdirectory

Reply:
ok